test/test_logger.rb

```diff
@@ -14,6 +14,16 @@ class TestLogger < Test::Unit::TestCase
     end
 
     context "#add" do
+      # logger.add(Logger::INFO, nil)
+      should 'implement add method with level, message and facility from defaults' do
+        @logger.expects(:notify_with_level!).with do |level, hash|
+          level == GELF::INFO &&
+          hash['short_message'] == 'gelf-rb' &&
+          hash['facility'] == 'gelf-rb'
+        end
+        @logger.add(GELF::INFO, nil)
+      end
```

Comment:
I added this test to explicitly highlight this behavior.
The [original idea](https://github.com/Graylog2/gelf-rb/commit/5a2c852e97743c18221dccfae032b741b527e643) was to have an consistent API with `Logger#add`, but it wasn't really the case. This could be refined I guess in a separate PR.